Makefile

```diff
@@ -16,21 +15,43 @@ KUSTOMIZE := $(BINDIR)/kustomize
 STATICCHECK := $(BINDIR)/staticcheck
 NILERR := $(BINDIR)/nilerr
 INEFFASSIGN := $(BINDIR)/ineffassign
-KUBEBUILDER_ASSETS := $(BINDIR)
 PROTOC := PATH=$(BINDIR):$(PATH) $(BINDIR)/protoc -I=$(PWD)/include:.
 PACKAGES := unzip lvm2 xfsprogs
+ENVTEST_ASSETS_DIR := testbin
 
 GO_FILES=$(shell find -name '*.go' -not -name '*_test.go')
 GOOS := $(shell go env GOOS)
 GOARCH := $(shell go env GOARCH)
 GO111MODULE = on
```

Comment:
`GO111MODULE = on` is no longer needed.
Modules on by default in Go 1.16.

Reply:
@llamerada-jp This comment seems not to be reflected.